### PR TITLE
Allow publishing a connector image via github comment

### PR DESCRIPTION
**Pull request by sherifnada**
## What
changes `manage.sh` to allow publishing a connector's docker image without running tests. The command `/publish`, which is supposed to be run after a successful `/test` run on a branch, will build and publish the connector's docker image. 

Depends on changes from #1899

### CODE REVIEW

File: tools/integrations/manage.sh [hunk 1]
```diff
@@ -45,7 +44,7 @@ cmd_publish() {
   echo "image_name $image_name"
   echo "$versioned_image $versioned_image"
   echo "latest_image $latest_image"
-
+
```
Comment (sherifnada) [inline tools/integrations/manage.sh:47]:
```suggestion

```

File: .github/workflows/test-command.yml [hunk 2]
```diff
@@ -85,9 +85,6 @@ jobs:
         id: test
         env:
           ACTION_RUN_ID: ${{github.run_id}}
-          BUILD_STAT_WRITE_KEY: ${{ secrets.BUILD_STAT_WRITE_KEY }}
```
Comment (sherifnada) [inline .github/workflows/test-command.yml:88]: these seemed to be unused

**Comment by sherifnada** [comment]
/publish connector=source-mailchimp branch=sherif/add-gh-publish-command

File: .github/workflows/test-command.yml (hunk 2 above)
Comment (jrhizor) [reply .github/workflows/test-command.yml:88]: forgot to remove them after switching to s3 for connector build history

File: tools/integrations/manage.sh [hunk 3]
```diff
@@ -17,25 +17,24 @@ cmd_build() {
...
 
   echo "Building $path"
-    ./gradlew "$(_get_rule_base "$path"):clean"
-    ./gradlew "$(_get_rule_base "$path"):build"
-    ./gradlew "$(_get_rule_base "$path"):integrationTest"
+  ./gradlew "$(_get_rule_base "$path"):clean"
+  ./gradlew "$(_get_rule_base "$path"):build"
 }
 
-_execute_task_if_exists() {
```
Comment (sherifnada) [inline tools/integrations/manage.sh:25]: this method was unused

**Review by cgardens — COMMENTED** [review]
Maybe I'm just confused but there's a few things i'm not understanding:
1. are we running publish after every successful run of test? if so why?
2. why does publish skip the tests https://github.com/airbytehq/airbyte/pull/1899? i'm not seeing where we get a guarantee that the tests ran successfully before publish runs.

**Comment by cgardens** [comment]
Doesn't this setup currently overwrite the previous version if someone merges to master without bumping the version? My understanding of the flow currently is
1. someone makes changes. does not bump connector version
1. integration tests run
1. we auto publish that version overwriting the previous version? or we just fail?

it seems like you're going for a continuous deploy setup? i was a little surprised because i thought you were just going for publishing via the github UI. i'm not necessarily against it, but i'm not sure some of the requisite checks are in place for this t work as intended. again forgive me if i'm just missing them. i have looked only at this PR and [1899](https://github.com/airbytehq/airbyte/pull/1899)

* can i run integration tests _without_ "deploying" if they succeed? i.e. if i do it from a branch.
* how do we make sure we don't accidentally overwrite previous versions of integrations
* do we have a guarantee that integration tests have always run before publish is called (or as part of publish)?
* is the name of the github publish action correct? https://github.com/airbytehq/airbyte/pull/1899/files#diff-ffc7fd15ed8fb1e93293af86d2a808b51d85f64089739fd94be2e3d72b59ea1bR1

**Comment by sherifnada** [comment]
With this PR, the expected flow is: 
1. Create a PR, iterate as much as you want, run `/test` as much as you want until satisfied. 
2. Bump connector version. 
3. run `/publish`

> how do we make sure we don't accidentally overwrite previous versions of integrations
`manage.sh` does not publish an image if it already exists on github, see [here](https://github.com/airbytehq/airbyte/blob/master/tools/integrations/manage.sh#L52)

**Comment by sherifnada** [comment]
> do we have a guarantee that integration tests have always run before publish is called (or as part of publish)?
with this change, it is possible to publish without running the test by passing `SKIP_TESTS=true` to `manage.sh`. 

The reason for decoupling those tests and publishing is that some integration tests take a long time to run (e.g 30m). Therefore, if I already ran them and they passed, it is safe to publish them. Otherwise, I'd need to `/publish` then wait for the tests to run again

**Review by michel-tricot — APPROVED** [review]

**Pull request merged**